### DISCUSSION TITLE
Pop-app API

### DISCUSSION THREAD

**Discussion by jfletcher20** [foivz/pop-app#12] [Announcements]
List of APIs for the application.

**Comment by mislavmatijevic** [comment]
Updated APIs on [Wiki](https://github.com/foivz/pop-app/wiki/APIs).

**Comment by mislavmatijevic** [comment]
Updated APIs on [Wiki](https://github.com/foivz/pop-app/wiki/APIs).
Fixed invoice generation and finalization APIs.
